Documentation/ExtensionArchitecture/FileStructure/ExtTables.rst

```diff
@@ -1,29 +1,32 @@
-.. include:: /Includes.rst.txt
-.. index:: ! File; EXT:{extkey}/ext_tables.php
-.. _ext-tables-php:
+..  include:: /Includes.rst.txt
+..  index:: ! File; EXT:{extkey}/ext_tables.php
+..  _ext-tables-php:
 
-======================
-:file:`ext_tables.php`
-======================
+================
+`ext_tables.php`
+================
 
-*-- optional*
+..  typo3:file:: ext_tables.php
+    :scope: extension
+    :regex: /^.*ext\_tables\.php/
```

Comment:
```suggestion
    :regex: /^.*ext\_tables\.php$/
```

To just match the file not something after? And should this match everything ending with ext_tables.php?

Reply:
Yes as it is the only file of that name within the typo3 universe and can be used in any kind of extension

Reply:
Good point! Will change that all over